src/main/java/com/dev/cinema/util/HibernateUtil.java

```diff
@@ -13,7 +14,7 @@ private static SessionFactory initSessionFactory() {
         try {
             return new Configuration().configure().buildSessionFactory();
         } catch (Exception e) {
-            throw new RuntimeException("Error creating Session Factory", e);
+            throw new DataProcessingException("Error creating Session Factory", e);
```

Comment:
```suggestion
            throw new RuntimeException("Error creating Session Factory", e);
```
this exception has a different nature